src/components.d.ts

```diff
@@ -11,7 +11,9 @@ import {
 } from './interface';
 
 export namespace Components {
-  interface ManifoldOauth {}
+  interface ManifoldOauth {
+    'tick'?: string;
```

Comment:
This was `tick` and not `key` because JSX/TypeScript has `key` as a reserved prop.